src/machines/systemIO/systemIOMachineDesktop.ts

```diff
@@ -204,7 +204,7 @@ export const systemIOMachineDesktop = systemIOMachine.provide({
 
         return {
           message: 'File created successfully',
-          fileName: input.requestedFileName,
+          fileName: newFileName,
```

Comment:
OPE nice catch

Reply:
yeee... I kinda want a better `<userInput>` to `<uniqueApplicationInput>` system I lost track of the conversions once or twice.